scala-package/infer/src/main/scala/ml/dmlc/mxnet/infer/ImageClassifier.scala

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ml.dmlc.mxnet.infer
+
+import ml.dmlc.mxnet._
+
+import scala.collection.mutable.ListBuffer
+
+// scalastyle:off
+import java.awt.image.BufferedImage
+// scalastyle:on
+import java.io.File
+import javax.imageio.ImageIO
+
+
+/**
+  * A class for classifier tasks
+  *
+  * @param modelPathPrefix  PathPrefix from where to load the symbol, parameters and synset.txt
+  *                         Example: file://model-dir/resnet-152(containing resnet-152-symbol.json
+  *                         file://model-dir/synset.txt
+  * @param inputDescriptors Descriptors defining the input node names, shape,
+  *                         layout and Type parameters
+  */
+class ImageClassifier(modelPathPrefix: String,
+                      inputDescriptors: IndexedSeq[DataDesc])
+                      extends Classifier(modelPathPrefix,
+                      inputDescriptors) {
+
+  val classifier: Classifier = getClassifier(modelPathPrefix, inputDescriptors)
+
+  // Loading image from file
+  def loadImageFromFile(inputImagePath: String): BufferedImage = {
+      val img = ImageIO.read(new File(inputImagePath))
+      img
+  }
+
+  /**
+    * Loading input batch of images
+    * @param inputImageDirPath
+    * @return List of buffered images
+    */
+  def loadInputBatch(inputImageDirPath: String): List[BufferedImage] = {
+    val dir = new File(inputImageDirPath)
+    val inputBatch = ListBuffer[BufferedImage]()
+    for (imgFile: File <- dir.listFiles()){
+      val img = ImageIO.read(imgFile)
+      inputBatch += img
+    }
+    inputBatch.toList
+  }
+
+  /**
+    * Reshape the input image to new shape
+    *
+    * @param img       image
+    * @param newWidth  rescale to new width
+    * @param newHeight rescale to new height
+    * @return Rescaled BufferedImage
+    */
+  def getScaledImage(img: BufferedImage, newWidth: Int, newHeight: Int): BufferedImage = {
```

Comment:
why not call this reshape Image? instead of getScaledImage. Also this can be static method, so users can just call this method without having to create an object. ie.., a method on the companion object